src/colorizer/utils/correlation.ts

```diff
@@ -6,7 +6,7 @@
  * A number between -1 and 1 representing the correlation. Returns 0 if there
  * are no finite values in the input arrays.
  */
-export function pearson(x: Float32Array, y: Float32Array): number {
+export function pearson(x: Float32Array | Uint32Array, y: Float32Array | Uint32Array): number {
```

Comment:
Ended up changing types in a few places to allow Uint32Array data, since the track + time data is stored as such.